pyramid_oereb/standard/create_standard_models.py

```diff
@@ -39,12 +48,21 @@ def create_standard_model():
         type='string',
         help='The schema name. If not specified, it will be derived from the code.'
     )
+    parser.add_option(
+        '-s', '--primary_key_is_string',
+        dest='primary_key_is_string',
+        metavar='PRIMARY KEY IS STRING',
+        help='Switch to choose if primary keys in desired new models are STRING or INTEGER'
+    )
+    primary_key_is_string = False
     options, args = parser.parse_args()
     if not options.code:
         parser.error('No oereb code set.')
     if not options.geometry_type:
         parser.error('No geometry_type set.')
     if not options.target_path:
         parser.error('No target_path set.')
+    if options.primary_key_is_string:
+        primary_key_is_string = True
     _create_standard_configuration_models_py_(options.code, options.geometry_type, options.target_path,
-                                              options.schema)
+                                              options.schema, primary_key_is_string)
```

Comment:
Why having a local variable we can directly use `str2bool(options.primary_key_is_string)`

Reply:
Old leftover. I changed it.

Reply:
Argparse support boolean variable, you can add `action='store_true'` in the add_option, an directly use `options.primary_key_is_string` here :-)